subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -1,3 +1,3 @@
 {
     "acceptedApiChanges": []
```

Comment:
This file here is auto-generated. So we'd need to fix the generation and not the files themselves.